### PR TITLE
Create Root, Stack and Tab Navigators for MVP

### PR DESCRIPTION
**Pull request by gilhsu**
Requires merge of `bump-fhir-kit-client`

### CODE REVIEW

File: src/screens/LoginScreen.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  View,
+  Text,
+  Image,
+  Button,
+} from 'react-native';
+import Login from '../components/Login/Login';
+import logo from '../../assets/images/vermonster-logo.png';
+
+const LoginScreen = ({ navigation }) => (
+  <SafeAreaView style={styles.safeAreaView}>
```
Comment (foxtree) [inline src/screens/LoginScreen.js:18]: The following documentation suggests that`SafeAreaView` might be iOS only -- is this accurate?
https://reactnative.dev/docs/safeareaview

>The purpose of SafeAreaView is to render content within the safe area boundaries of a device. It is currently only applicable to iOS devices with iOS version 11 or later.

**Comment by foxtree** [comment]
Under some conditions, I am seeing un-handled errors, e.g., clicking "Cancel"

image


These may or may not show up in a production build, but: I'm wondering if a handler can be wired-up (if not already), even if there's just an empty function body.

**Comment by foxtree** [comment]
Similarly, when clicking "Deny"
image

**Review by foxtree — APPROVED** [review]

File: src/screens/LoginScreen.js (hunk 1 above)
Comment (gilhsu) [reply src/screens/LoginScreen.js:18]: Interestingly, android does not have an elegant solution for this. Instead you need to import `StatusBar` and measure the height.

**Pull request merged**